pybutton/client.py

```diff
@@ -32,25 +32,29 @@ class Client(object):
 
     '''
 
-    def __init__(self, api_key, config={}):
+    def __init__(self, api_key, config=None):
 
         if not api_key:
             raise ButtonClientError((
                 'Must provide a Button API key.  Find yours at'
                 ' https://app.usebutton.com/settings/organization'
             ))
 
-        config = self._config_with_defaults(config)
+        if config is None:
+            config = {}
+
+        config = _config_with_defaults(config)
 
         self.orders = Orders(api_key, config)
 
-    def _config_with_defaults(self, config):
-        secure = config.get('secure', True)
-        defaultPort = 443 if secure else 80
 
-        return {
-            'secure': secure,
-            'timeout': config.get('timeout'),
-            'hostname': config.get('hostname', 'api.usebutton.com'),
-            'port': config.get('port', defaultPort)
-        }
+def _config_with_defaults(config):
+    secure = config.get('secure', True)
+    defaultPort = 443 if secure else 80
+
+    return {
+        'secure': secure,
+        'timeout': config.get('timeout'),
+        'hostname': config.get('hostname', 'api.usebutton.com'),
+        'port': config.get('port', defaultPort)
```

Comment:
still need to add trailing commas!  I've seen some in README / tests we should make sure get them as well.